packages/api/src/external/carequality/api.ts

```diff
@@ -13,3 +14,20 @@ export function makeCarequalityAPI(apiKey?: string): Carequality | undefined {
   const cqApiKey = apiKey ?? Config.getCQApiKey();
   return new Carequality(cqApiKey);
 }
+
+/**
+ * Creates a new instance of the IHE Gateway client.
+ * @returns IHE Gateway client.
+ */
+export function makeIheGatewayAPI(): IHEGateway {
+  // TODO ADD IHE GATEWAY MOCK
+  // if (Config.isSandbox()) {
+  //   return;
+  // }
+
```

Comment:
When we start to handle the IHE Gateway in the cloud then we can add worry about adding this

Reply:
Well because it wont only be disabling it. I assumed that we would want to mock it the same way we do with CW unless for sandbox having the mock payloads just from CW is enough for our cx's.

Reply:
I guess Raf's point is why we don't sure add the piping for the mock now, so we don't have to do it later? 

Like https://github.com/metriport/metriport/blob/baf47279dada489d0d0ced37ef5dfd8cbb2710c5/packages/api/src/external/commonwell/mock.ts#L23